src/styled-components/theme/theme.styled.ts

```diff
@@ -0,0 +1,40 @@
+import {DefaultTheme} from 'styled-components/native';
+
+declare module 'styled-components/native' {
+  export interface DefaultTheme {
+    colors: {
+      [key: string]: string;
```

Comment:
This type isn't goint to allow you to use suggestions to select a color, you can use `typeof defaultTheme`

Reply:
You can keep using this, not problem.